quiztest/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/inventory
+spring.datasource.username=postgres
+spring.datasource.password=1horsePlease!
```

Comment:
Fine for test applications, but just be aware your database password is exposed here in plaintext